EKF/estimator_interface.h

```diff
@@ -265,8 +265,8 @@ class EstimatorInterface
 	// return true if the global position estimate is valid
 	virtual bool global_position_is_valid() = 0;
 
-	// return true if the EKF is dead reckoning the position using inertial data only
-	bool inertial_dead_reckoning() {return _is_dead_reckoning;}
+	// return true if the EKF has exceeded the maximum time for dead reckoning the position using inertial data only
+	bool inertial_dead_reckoning() {return _deadreckon_time_exceeded;}
```

Comment:
This is quite a functional change. This seems not related to the rest of the PR. Could be good to have this as a separate PR.

Reply:
Will remove. With improvements to the internal reset we are no longer relying on the commander to trigger a reset externally.

Reply:
See https://github.com/PX4/ecl/pull/751/commits/8606d981dd3c1d34e6432cd6036af8e45d8650a5